.travis.yml

```diff
@@ -20,27 +25,20 @@ before_script:
     export PATH=$HOME/.local/bin:$PATH
 
 script:
-# TEMPORARY UNTIL SERDE 0.7 IS RELEASED
-- |
-  mkdir .travis-deps
-  mkdir .cargo
-  echo 'paths = ["./.travis-deps/serde"]' > .cargo/config
-  git clone https://github.com/serde-rs/serde .travis-deps/serde
-  pushd .travis-deps/serde
-  git reset --hard origin/master
-  popd
 - |
-    (cd json && travis-cargo build) &&
-    (cd json && travis-cargo test) &&
-    (cd json_tests && travis-cargo test -- --features with-syntex --no-default-features) &&
-    (cd json_tests && travis-cargo --only nightly test -- features nightly-testing) &&
-    (cd json_tests && travis-cargo --only nightly bench -- features nightly-testing) &&
-    (cd json_tests && travis-cargo --only nightly bench -- --features "nightly-testing with-syntex" --no-default-features) &&
-    (cd json && travis-cargo --only stable doc)
+    if [ -z $BENCH ]; then
```

Comment:
`-z` means empty i.e. `BENCH.len() == 0`. We run four builds: the three release channels stable beta and nightly, and a fourth one that is nightly with an environment variable BENCH=1. The bench build was the longest so I split it out into it's own build to increase parallelism. The stable/beta/nightly builds all run the first part of this `if` statement. The BENCH=1 build runs the second part which is just `travis-cargo bench`.